servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/WriteDemandEstimators.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2020 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.transport.netty.internal;
+
+/**
+ * Utility methods associated with {@link WriteDemandEstimator}.
+ */
+public final class WriteDemandEstimators {
+    private WriteDemandEstimators() {
+        // no instance
+    }
+
+    /**
+     * Returns a new instance of a default implementation of {@link WriteDemandEstimator}.
+     *
+     * @return A new instance of a default implementation of {@link WriteDemandEstimator}.
+     */
+    public static WriteDemandEstimator newDefaultEstimator() {
+        return new MaxSizeBasedWriteDemandEstimator();
```

Comment:
Since we have only a single implementation of `WriteDemandEstimator` atm, and both classes (this and `MaxSizeBasedWriteDemandEstimator`) are in the same `io.servicetalk.transport.netty.internal` package, can we just make `MaxSizeBasedWriteDemandEstimator` public instead of introducing new factory? Because the code is internal, we can add it later if necessary.